storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -2364,4 +2374,23 @@ public static long bitXor(Long a, Long b) {
         return rtn;
     }
 
+    /**
+     * converts a clojure PersistentMap to java HashMap
+     */
+    public static Map<String, Object> convertMap(Map map) {
```

Comment:
It might be better to rename to `convertClojureMapToJavaMap` or others which can represent from and to.
If we can specialize parameter's type it would be better.